integration/data_flow.md

```diff
@@ -0,0 +1,109 @@
+# Observatory Data Flow
+
+For users who wish to utilize the [Science Archive](https://github.com/observatorycontrolsystem/science-archive) for data archival and retrieval, this document describes the components necessary.
+
+## Main Components
+
+* [OCS Ingester](https://github.com/observatorycontrolsystem/ingester) - commonly used as a library to simplify the ingestion process of observatory data products into the science archive. 
+* [OCS Science Archive](https://github.com/observatorycontrolsystem/science-archive) - A Django application which aids in the archival storage and retrieval of observatory data products.
+
+Both of these applications make use of a common library, the [OCS Archive Library](https://github.com/observatorycontrolsystem/ocs_archive). The OCS Archive library allows the OCS Ingester and Science Archive to share common configuration and reduces the need for duplicated code across the two projects - this shared code contains classes used to abstract data files and data storage backends. See the "Advanced Topics" for more information about customizing the behavior of the `ocs_archive` library.
+
+
+## Configuration
+
+Because both the OCS Science Archive and OCS Ingester depend on the OCS Archive Library, the two can share common configuration. Runtime configuration for the OCS Archive Library is set using environment variables - see [OCS Archive Configuration Documentation](https://github.com/observatorycontrolsystem/ocs_archive#configuration) for more detail about possible configuration options.
+
+## Telescope Control System Integration
+
+In the execution of an observation, the observatory's Telescope Control System generates data products. For these products to be archived and served to users or staff via an OCS Science Archive, they must be ingested.
+
+### A Simple Ingestion Scheme
+
+Once a TCS has created its final data products to be archived, the TCS may call out to the Ingester library directly. If the TCS is written in Python, the call is as simple as:
```

Comment:
"the call is as simple as" -> "the call is as simple as setting the relevant environment variables for your filestore type and running" or something like that, just to hammer in that environment variables must be setup for this library to work properly